WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -184,14 +184,20 @@ public async Task<bool> StartRoundAsync(IEnumerable<SmartCoin> smartCoins, Round
 
 			// Output registration.
 			roundState = await RoundStatusUpdater.CreateRoundAwaiter(roundState.Id, Phase.OutputRegistration, cancellationToken).ConfigureAwait(false);
-			Logger.LogDebug($"Round ({roundState.Id}): Output registration phase started.");
 			Destinations = destinations;
 
-			await scheduler.StartOutputRegistrationsAsync(outputTxOuts, bobClient, KeyChain, cancellationToken).ConfigureAwait(false);
+			var remainingTime = roundState.OutputRegistrationTimeout - RoundStatusUpdater.Period;
+			var outputRegistrationEndTime = DateTimeOffset.UtcNow + (remainingTime * 0.8);
+			Logger.LogDebug($"Round ({roundState.Id}): Output registration phase started - it will end in: {outputRegistrationEndTime - DateTimeOffset.UtcNow:hh\\:mm\\:ss}");
```

Comment:
The log entry is confusing because it uses `OutputRegistration Phase` but that's a concept is different than the `OutputRegistration Phase` on the server side. 

Just removing the word `phase` would be better.

Reply:
Done.